### ISSUE TITLE
EDNS unknown version: query not in response

### ISSUE THREAD

**Issue by AngeloColucci** [NLnetLabs/nsd#208]
Hi @wcawijngaards 
I'm experimenting the same issue in a similar configuration scenario described in the issue https://github.com/NLnetLabs/nsd/issues/97 (dnsdist 1.7.0 in front of nsd 4.2.4 epel package). 
When I try to exectute dnsflag day 2020 auth nameserver validation I receive minor warning on the query:

dig +edns=1 soa zone @server

Capturing the network traffic i noticed that in the query response is missing the initial request.
Does this nsd version is fixable?

Thanks in advance for your support
Angelo

**Comment by wcawijngaards** [comment]
To fix it you need to upgrade, the fix for that issue appears in NSD 4.3.2. When I test the latest version in the code repository, that version seems to work fine too. So an upgrade, to perhaps the latest version, of NSD would solve the issue.

**Comment by AngeloColucci** [comment]
Hi @wcawijngaards,
thanks for your update. 
So I need to request that version to the mantainer of epel repository, because they still promote the 4.2.4.

BR
Angelo

**Comment by wcawijngaards** [comment]
Yes, we do not maintain those repositories. You could also compile NSD manually, from source code. That creates a version that installs in /usr/local by default.
